src/_components/card/card.css

```diff
@@ -158,17 +158,17 @@ a.card-mode:focus {
   padding-bottom: 0;
 }
 .card-mode .card-media .card-image {
-  border-top-left-radius: 10px;
-  border-top-right-radius: 10px;
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
```

Comment:
det skal vel ikke være hardcoded?

Uanset må du meget gerne reverte dine ændringer i den fil - så slipper jeg for kæmpe merge konflikt om lidt når det meste er ændret alligevel